clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -0,0 +1,276 @@
+        }
+      } catch (IOException e) {
+        LOGGER.error("Error in closing reader for command: {}", this, e);
+      }
+    }
+
+    if (params.debug) {
+      LOGGER.info("Cmd: {}, StdOut: {}, Exit code: {}", this, stdOut, exitCode);
+    }
+  }
+
+  @Override
+  public String toString() {
+    String cmd = "StoreRepushCommand{\n" + String.join(" ", shellCmd) + "\n}";
+    // Hide user's password.
+    cmd = cmd.replace(params.getPassword(), "******");
+    return cmd;
+  }
+
+  public static class Params {
+    // command name;
+    private String command;
+    // extra arguments to command.
+    private String extraCommandArgs;
+    // Password + VIP.
+    private String password;
+    // Debug run.
+    private boolean debug = false;
+
+    public void setPassword(String password) {
+      this.password = password;
+    }
+
+    public String getPassword() {
+      return this.password;
+    }
+
+    public void setDebug(boolean debug) {
+      this.debug = debug;
+    }
+
+    public void setCommand(String cmd) {
+      this.command = cmd;
+    }
+
+    public void setExtraCommandArgs(String args) {
+      this.extraCommandArgs = args;
+    }
+  }
+
+  public static class Result {
+    private String cluster;
+    private String store;
+    private String stdOut;
+    private int exitCode;
+    private String error;
+    private String message;
+
+    public int getExitCode() {
+      return exitCode;
+    }
+
+    public void setExitCode(int exitCode) {
+      this.exitCode = exitCode;
+    }
+
+    public String getStdOut() {
+      return stdOut;
+    }
+
+    public void setStdOut(String stdOut) {
+      this.stdOut = stdOut;
+    }
+
+    public boolean isError() {
+      return error != null;
+    }
+
+    public String getCluster() {
+      return cluster;
+    }
+
+    public void setCluster(String cluster) {
+      this.cluster = cluster;
+    }
+
+    public String getStore() {
+      return store;
+    }
+
+    public void setStore(String store) {
+      this.store = store;
+    }
+
+    public void setError(String error) {
+      this.error = error;
+    }
+
+    public String getError() {
+      return error;
+    }
+
+    public String getMessage() {
+      return message;
+    }
+
+    public void setMessage(String message) {
+      this.message = message;
+    }
+
+    public void parseStandardOutput() {
+      // No standard output or empty output.
+      if (stdOut == null || stdOut.equals(StringUtils.EMPTY)) {
+        return;
+      }
+
+      // Command reached to Azkaban, no matter it was a success or a failure.
+      if (matchSuccessPattern()) {
+        return;
+      }
+
+      if (matchFailurePattern()) {
+        return;
+      }
+
+      // Failed: repush command itself hit an error (e.g. incomplete parameters)
+      error = stdOut;
+    }
+
+    private boolean matchSuccessPattern() {
+      // success: https://example.com/executor?execid=21585379
+      String successPattern = "^success:(.*)$";
```

Comment:
I think the regex pattern should handle for the `space` after `success:`? Same for `failure:` below

Reply:
Makes sense. Will fix it.